app/components/public/ticket-list.js

```diff
@@ -40,6 +40,24 @@ export default Component.extend(FormMixin, {
     ) > 0;
   }),
 
+  donationTickets: computed('data', function() {
+    return this.data.filterBy('type', 'donation');
+  }),
+
+  isDonationPriceValid: computed('donationTickets', 'donationTickets.@each.orderQuantity', 'donationTickets.@each.price', function() {
+    let returnValue = false;
+    this.donationTickets.forEach(donationTicket => {
+      if (donationTicket.orderQuantity >= 0) {
+        if (donationTicket.price > 0) {
```

Comment:
I've added `donationTickets` to the watchable properties of the computed property. Do I need to explicitly resolve this again by using `donationTicket.get('orderQuantity')`? Will this be returning a promise here `donationTicket.orderQuantity`? @pradeepgangwar @shreyanshdwivedi @uds5501 @CosmicCoder96